examples/ucp_client_server.c

```diff
@@ -385,22 +336,41 @@ static int request_finalize(ucp_worker_h ucp_worker, test_req_t *request,
     /* Print the output of the first, last and every PRINT_INTERVAL iteration */
     if ((current_iter == 0) || (current_iter == (num_iterations - 1)) ||
         !((current_iter + 1) % (PRINT_INTERVAL))) {
-        msg_str = copy_buffer(iov);
-        if (msg_str == NULL) {
-            fprintf(stderr, "memory allocation failed\n");
-            ret = -1;
-            goto release_iov;
-        }
-
-        print_result(is_server, msg_str, current_iter);
-        free_copied_buffer(msg_str);
+        print_result(is_server, iov, current_iter);
     }
 
 release_iov:
     buffer_free(iov);
     return ret;
 }
 
+static
+ucs_status_t setup_send_recv_ctx(ucp_dt_iov_t *iov, int is_client,
```

Comment:
1. return int
2. rename to "fill_request_param"

Reply:
Done